server/filestore_test.go

```diff
@@ -10620,6 +10620,67 @@ func TestFileStoreCacheLookupOnEmptyBlock(t *testing.T) {
 	})
 }
 
+func TestFileStoreCachePartialError(t *testing.T) {
+	testFileStoreAllPermutations(t, func(t *testing.T, fcfg FileStoreConfig) {
+		// fcfg.BlockSize = 40
+		fcfg.SyncAlways = true
+		fcfg.AsyncFlush = false
+		cfg := StreamConfig{Name: "zzz", Subjects: []string{"foo"}, Storage: FileStorage}
+		created := time.Now()
+		fs, err := newFileStoreWithCreated(fcfg, cfg, created, prf(&fcfg), nil)
+		require_NoError(t, err)
+		defer fs.Stop()
+
+		for n := range 10 {
+			fmt.Println("Storing", n+1)
+			_, _, err = fs.StoreMsg("test", nil, []byte{1, 2, 3, 4, 5}, 0)
+			require_NoError(t, err)
+		}
+
+		_, err = fs.EraseMsg(4)
+		require_NoError(t, err)
+		_, err = fs.EraseMsg(1)
+		require_NoError(t, err)
+		_, err = fs.EraseMsg(2)
+		require_NoError(t, err)
+		// _, err = fs.RemoveMsg(3)
+		// require_NoError(t, err)
+		_, err = fs.EraseMsg(5)
+		require_NoError(t, err)
+
+		// _, err = fs.Compact(2)
```

Comment:
Can these commented-out lines be removed?

Reply:
Actually that test shouldn't have been committed at all.